### PR TITLE
Fixed #1459 -- Fixed get_all_doc_versions template tag.

### LINKED ISSUE #1459: Site version picker rendering link text as tuples with identical `href` attributes; stuck on 5.0 without URL changes
I've never seen an issue with the Django docs site version picker before, but nothing is reachable besides 5.0 without manually changing the URL.

I think I've found a potential candidate, involving the templatetag: https://github.com/django/djangoproject.com/commit/7596a1ffa861702948a30ec3fb27d686bdb47209#r136155462

@felixxm Hi, sorry to tag you if I'm wrong, but it's the only recent change that seems relevant :)

Reproduced on Safari to validate my theory.

### PR DESCRIPTION
**Pull request by felixxm**
Regression in 7596a1ffa861702948a30ec3fb27d686bdb47209.

### CODE REVIEW

File: docs/tests.py [hunk 1]
```diff
@@ -195,9 +197,17 @@ def test_empty_get(self):
 class TemplateTagTests(TestCase):
     fixtures = ["doc_test_fixtures"]
 
-    def test_get_all_doc_versions(self):
+    def test_get_all_doc_versions_empty(self):
         with self.assertNumQueries(1):
-            get_all_doc_versions({})
+            self.assertEqual(get_all_doc_versions({}), ["dev"])
+
+    def test_get_all_doc_version(self):
```
Comment (nessita) [inline docs/tests.py:204]:
```suggestion
    def test_get_all_doc_versions(self):
```

**Review by nessita — APPROVED** [review]
LGTM, thanks! Added minor suggestion.

File: docs/tests.py (hunk 1 above)
Comment (felixxm) [reply docs/tests.py]: Typo, fixed.

**Comment by felixxm** [comment]
@nessita Thanks :+1:

**Comment by trojjer** [comment]
Awesome :)

**Pull request merged**